src/app/client/src/app/modules/notes/components/inline-editor/inline-editor.component.ts

```diff
@@ -163,26 +163,31 @@ export class InlineEditorComponent implements OnInit, AfterViewInit, OnDestroy {
         }
       };
       this.noteService.create(requestData).pipe(
-      takeUntil(this.unsubscribe$))
-      .subscribe(
-        (apiResponse: ServerResponse) => {
-          const returnObj = {
-            note: requestData.request.note,
-            userId: requestData.request.userId,
-            title: requestData.request.title,
-            courseId: requestData.request.courseId,
-            contentId: requestData.request.contentId,
-            createdBy: requestData.request.createdBy,
-            updatedBy: requestData.request.updatedBy,
-            createdDate: new Date().toISOString(),
-            updatedDate: new Date().toISOString()
-          };
-          this.createEventEmitter.emit(returnObj);
-        },
-        (err) => {
-          this.toasterService.error(this.resourceService.messages.fmsg.m0030);
-        }
-      );
+        takeUntil(this.unsubscribe$))
+        .subscribe(
+          (apiResponse: ServerResponse) => {
+            if (apiResponse.result.id) {
+              const returnObj = {
```

Comment:
move this returnObj to service and get from there as this code is replicated in popup-editor-component